### PR TITLE
Improve HeatmapControl and batchLoader Documentation

### PR DESCRIPTION
**Pull request by AkhileshAdithya**
## Description
- Improved in-code documentation to heatmapcontrol.js and batchloader.js 

## Motivation and Context
**Why is this change required? What problem does it solve?
- If it fixes an open issue, please link to the issue here.
- helps in #413 

## How Has This Been Tested?
**Please describe in detail how you tested your changes.
- Doesn't affect code

## Types of changes
**What types of changes does your code introduce? Put an `x` in all the boxes that apply:
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:
**Go over all the following points, and put an `x` in all the boxes that apply.
**(If you're unsure about any of these, don't hesitate to ask. We're here to help!)
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by birm — APPROVED** [review]
Looks reasonable to me, thanks for having a look at the docs.

**Comment by AkhileshAdithya** [comment]
@birm, I've added some documentation for the batchLoader.js file. Check it out and lemme know if there needs to be any changes

File: components/heatmapcontrol/heatmapcontrol.js [hunk 1]
```diff
@@ -277,6 +285,11 @@ HeatmapControl.prototype.__opacityChange = function() {
   }
 };
 
+// createSelect         - creates options for each field
+// createField          - creates fields and their sliders
+// createOpacities      - creates opacity field and it's slider
+// createIntervalInputs - creates HTML Color inputs for given noOfIntervals
```
Comment (birm) [inline components/heatmapcontrol/heatmapcontrol.js:291]: Can you put these right before their appropriate functions?

Comment (AkhileshAdithya) [reply components/heatmapcontrol/heatmapcontrol.js]: Done

**Pull request merged**